packages/docs/site/docs/main/guides/for-theme-developers.md

```diff
@@ -198,6 +198,25 @@ With the [`importThemeStarterContent` step](/blueprints/steps#importThemeStarter
 
 You can also publish the starter content of a theme when installing it with the [`installTheme` step](/blueprints/steps#installTheme) by setting to `true` its `importStarterContent` option.
 
+#### `wp-cli`
+
+Another way of generating content for your theme is via the `wp-cli` step that allows you to run [WP-CLI commands](https://developer.wordpress.org/cli/commands/) such as [`wp post generate`](https://developer.wordpress.org/cli/commands/post/generate/):
+
+```json
+{
+	"landingPage": "/wp-admin/edit.php",
+	"login": true,
+	"steps": [
+		{
+			"step": "wp-cli",
+			"command": "wp post generate --count=20 --post_type=post --post_date=1999-01-04"
+		}
+	]
+}
+```
+
+[<kbd> &nbsp; Run Blueprint &nbsp; </kbd>](https://playground.wordpress.net/builder/builder.html#{%22landingPage%22:%22/wp-admin/edit.php%22,%22login%22:true,%22steps%22:[{%22step%22:%22wp-cli%22,%22command%22:%22wp%20post%20generate%20--count=20%20--post_type=post%20--post_date=1999-01-04%22}]})
+
```

Comment:
You could also link to the Blueprint Gallery entry of https://github.com/WordPress/blueprints/tree/trunk/blueprints/wpcli-post-with-image
The shows the connection between content and featured image.

Reply:
I added a tip referencing [that example ](https://github.com/WordPress/wordpress-playground/pull/1747/commits/a4110b622e3653189dbcba6569fb619086183fb3)